app/controllers/user_profiles_controller.rb

```diff
@@ -25,8 +27,10 @@ def update
     end
   end
 
-  def destroy
-    @user_profile.destroy
+  def stats_data
+    @individual_stats_presenter = IndividualStatisticsPresenter.new(user: @user)
+    @courses_list = @user.courses.where('courses_users.role = ?', CoursesUsers::Roles::INSTRUCTOR_ROLE)
+    @courses_presenter = CoursesPresenter.new(current_user: current_user, courses_list: @courses_list)
```

Comment:
Hmm... with this change, it means that we have to do the pretty slow queries of `CoursesPresenter` twice; once for the html view, and another for the json stats data.

I think that's okay for now, but eventually we'll want to move the courses list to React as well.